events/equeue/tests/tests.c

```diff
@@ -1,835 +0,0 @@
-/*
```

Comment:
I'm not sure if it's good idea to remove this test file as it's part of equeue library and still runs on CI (https://github.com/ARMmbed/mbed-os/blob/master/.travis.yml#L230). It should be left as is and only disable on travis script.